### PR TITLE
[pipeline] Add trigger:mender-dist-packages job

### PR DESCRIPTION
**Pull request by lluiscampos**
To trigger a build of the APT repository on changes in mender-connect.

### CODE REVIEW

**Comment by lluiscampos** [comment]
Stopping the mender-qa pipelines(s) as this won't be verified there.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -34,3 +35,18 @@ build:make:
     - apk add --update git make gcc pkgconfig libc-dev glib-dev
   script:
     - make build
+
+trigger:mender-dist-packages:
+  image: alpine
+  stage: trigger
+  before_script:
+    - apk add --no-cache curl git
```
Comment (tranchitella) [inline .gitlab-ci.yml:43]: Why do you need git?

**Review by tranchitella — APPROVED** [review]
I verified the URL, the syntax, all looks okay

File: .gitlab-ci.yml (hunk 1 above)
Comment (lluiscampos) [reply .gitlab-ci.yml:43]: I don't need it, that was a left over from the `mender` pipeline I copied from. Thanks!

**Pull request merged**